utilities/getPageMetadata.ts

```diff
@@ -65,10 +72,7 @@ export const getPageMetadata = ({
   const twitterCropped = getProcessedImage(twitter_image, '1200x600');
   const ogImage = ogCropped || heroImageCropped;
 
-  let ogType = 'website';
-  if (slug.startsWith('stories/')) {
-    ogType = 'article';
-  }
+  const ogType = component === 'sbStoryMvp' ? 'article' : 'website';
```

Comment:
Use top level component as check for og type. Basically if it's a story, use type "article", otherwise just "website"